### PR TITLE
feat(www): Added toggle buttons to change layout of component guide site.

### PR DESCRIPTION
**Pull request by mlipardo**
Change Article to a react component and added toggle button to change layout.

### CODE REVIEW

File: libs/docs/src/Articles/Article.jsx [hunk 1]
```diff
@@ -1,20 +1,48 @@
...
+        title={title}
+        layout={this.state.layout}
+        Breadcrumb={
+          <Breadcrumb items={[{ title: 'Home', path: '/' }, ...breadcrumbs]} />
+        }
+        main={main}
+        sidenav={<SideNav routes={sidebar} />}
+        PageActions={
+          <ButtonGroup>
+            <Button
```
Comment (eckdanny-osi) [inline libs/docs/src/Articles/Article.jsx]: could add `active: boolean` and `aria-pressed: booloean`  on the `Buttons`

Comment (eckdanny-osi) [inline libs/docs/src/Articles/Article.jsx]: could add `active: boolean` and `aria-pressed: booloean`  on the `Buttons`

File: libs/docs/src/Articles/Article.jsx [hunk 2]
```diff
@@ -1,20 +1,48 @@
...
+              color="primary"
+              size="sm"
+              onClick={() => this.setState({ layout: 'subroutes' })}
```
Comment (eckdanny-osi) [inline libs/docs/src/Articles/Article.jsx]: could add a class method `handleToggleClick` so no inline anonymous fns

File: libs/docs/src/Articles/Article.test.js [hunk 3]
```diff
@@ -1,11 +1,42 @@
...
+  describe('changes the layout', () => {
+    const wrapper = mount(<Article />)
+
+    it('initialize as subroutes', () => {
+      expect(wrapper.state('layout')).toEqual('subroutes')
+      expect(wrapper.find(SideNav).length).toBe(1)
+    })
+    it('from subroutes to dashboard', () => {
+      wrapper
+        .find(Button)
```
Comment (eckdanny-osi) [inline libs/docs/src/Articles/Article.test.js]: selecting from DOM by aria context or innerText would be better than order so test is less brittle

**Comment by SergiiRedchuk** [comment]
retest this please

File: libs/docs/src/Articles/Article.jsx (hunk 1 above)
Comment (mlipardo) [reply libs/docs/src/Articles/Article.jsx]: added the attributes. 👍

File: libs/docs/src/Articles/Article.jsx (hunk 2 above)
Comment (mlipardo) [reply libs/docs/src/Articles/Article.jsx]: added a class method.

**Pull request merged**